contracts/extension/plugin/Map.sol

```diff
@@ -3,61 +3,103 @@ pragma solidity ^0.8.0;
 
 import "../interface/plugin/IMap.sol";
 
-import "../Multicall.sol";
 import "../../openzeppelin-presets/utils/EnumerableSet.sol";
 
 /**
  *  TODO:
  *      - Remove OZ EnumerableSet external dependency.
  */
 
-contract Map is IMap, Multicall {
+abstract contract Map is IMap {
```

Comment:
Add comments demarcating different sections of the contract like in e.g. `Router`.